lib/language.js

```diff
@@ -54,10 +54,14 @@ exports.errors = {
         length: 'must be {{limit}} bytes'
     },
     date: {
-        base: 'must be a number of milliseconds or valid date string',
+        base: 'must be a valid date string or timestamp',
```

Comment:
Maybe `timestamp.javascript` should not even exist since I think it should be the same error message.